<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="guide_link_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>User guide</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>811</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>-50</y>
      <width>20</width>
      <height>681</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="f1_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>f1</string>
    </property>
   </widget>
   <widget class="QLabel" name="f2_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>f2</string>
    </property>
   </widget>
   <widget class="QLabel" name="f3_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>f3</string>
    </property>
   </widget>
   <widget class="QLabel" name="choose_method_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose the method</string>
    </property>
   </widget>
   <widget class="QLabel" name="rect_left_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangles (left)</string>
    </property>
   </widget>
   <widget class="QLabel" name="rect_mid_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangles (middle)</string>
    </property>
   </widget>
   <widget class="QLabel" name="rect_right_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectangles (right)</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="eq_group_box">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>60</y>
      <width>41</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="f1_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="f2_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="f3_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="f4_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="f5_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="method_group_box">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>300</y>
      <width>41</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="rect_left_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="rect_mid_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="rect_right_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="trap_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QRadioButton" name="simpson_button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="a_line_edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="a_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>a</string>
    </property>
   </widget>
   <widget class="QLabel" name="b_label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>b</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b_line_edit">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>440</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="solve_button">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>540</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>solve</string>
    </property>
   </widget>
   <widget class="QLabel" name="eps_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>440</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>eps</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eps_line_edit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>440</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="f4_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>f4</string>
    </property>
   </widget>
   <widget class="QLabel" name="f5_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>f5</string>
    </property>
   </widget>
   <widget class="QLabel" name="trap_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Trapezoidal</string>
    </property>
   </widget>
   <widget class="QLabel" name="simpson_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Simpson's</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="history_table">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>361</y>
      <width>481</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="plot_label">
    <property name="geometry">
     <rect>
      <x>309</x>
      <y>45</y>
      <width>481</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string>  </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
